akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/StashManagement.scala

```diff
@@ -45,12 +45,12 @@ private[akka] trait StashManagement[C, E, S] {
       case e: StashOverflowException =>
         setup.settings.stashOverflowStrategy match {
           case StashOverflowStrategy.Drop =>
-            if (context.log.isWarningEnabled) {
+            if (context.log.isWarnEnabled) {
```

Comment:
Weird optimization for something probably everyone has enabled.

Reply:
ok, removed since it didn't look too costly to do the extra work